modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/DefaultExtensionDesignTimeResolversFactory.java

```diff
@@ -265,6 +267,16 @@ public MetadataContext createMetadataContext(Supplier<Optional<ConfigurationInst
     return new DefaultMetadataContext(configurationSupplier, connectionManager, cache, typeLoader);
   }
 
+  @Override
+  public MetadataContext createMetadataContext(Supplier<Optional<ConfigurationInstance>> configurationSupplier,
```

Comment:
approving for now but need to revisit this class and the commit that introduced it (see conversation in mule-framework channel)

Reply:
Checked, and agreed. Previous to my rebase I didn't have this :/